include/sos/EnvelopeProblemSOS.tpp

```diff
@@ -0,0 +1,506 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
this is an application of the SOS solver, right? Could it be an example instead (placed for instance in  `examples/EnvelopeProblemSOS` directory) since it has a lot of unfinished parts and TODOs ?